src/store/index.ts

```diff
@@ -1,5 +1,4 @@
 import { createLogger, createStore } from "vuex";
-import tokenFile from "./token.json";
```

Comment:
We should not remove this import since it is necessary currently for the attestation workaround if the server is not running.

Reply:
Cool, I just removed this to keep eslint clean.

Reply:
Cool, I restored the local token fall-back in a helper that will automatically fall back with a short (interactive-appropriate) timeout in development mode. How does that look?